src/api/cow/types.ts

```diff
@@ -1,6 +1,6 @@
 import { OrderKind } from '@gnosis.pm/gp-v2-contracts'
-import { SupportedChainId as ChainId } from '/constants/chains'
-import { OrderCancellation, SigningSchemeValue } from '/utils/sign'
+import { SupportedChainId as ChainId } from '../../constants/chains'
+import { OrderCancellation, SigningSchemeValue } from '../../utils/sign'
```

Comment:
Does it work if you use absolute paths instead? I mean, like the other changes, just removing the leading `/`